### PR TITLE
supportedJujuSeries() must include EMS and non Ubuntu juju supported series.

### PR DESCRIPTION
**Pull request by hmlanigan**
## Description of change

For the juju deploy command, supportedJujuSeries() must include EMS,  Ubuntu juju supported series and non Ubuntu juju supported series, otherwise juju deploy to series such as Disco or centos7 will require --force .

Requires: https://github.com/juju/os/pull/7

## QA steps

1. Deploy  acceptancetests/repository/charms-centos/dummy-source, after adding to metadata.yaml:
```
series:
  - centos7
```
2. Deploy acceptancetests/repository/charms/dummy-source --series disco, it maybe necessary to add to metadata.yaml:
```
series:
  - disco
```
3. Other deploys should work as expected and per https://github.com/juju/juju/pull/10125.

### CODE REVIEW

File: cmd/juju/application/deploy_test.go [hunk 1]
```diff
@@ -948,6 +950,49 @@ func (s *DeploySuite) TestDeployLocalWithSeriesAndForce(c *gc.C) {
...
+func (s *DeploySuite) TestDeployLocalWithSupportedNonESMSeries(c *gc.C) {
+	nonEMSSeries, loggingPath := s.setupNonESMSeries(c)
+	_, _, err := s.runDeployWithOutput(c, loggingPath, "--series", nonEMSSeries)
+	c.Assert(err, jc.ErrorIsNil)
+}
+
+func (s *DeploySuite) TestDeployLocalWithNotSupportedNonESMSeries(c *gc.C) {
+	_, loggingPath := s.setupNonESMSeries(c)
+	_, _, err := s.runDeployWithOutput(c, loggingPath, "--series", "artful")
+	c.Assert(err, gc.ErrorMatches, "logging is not available on the following series: artful not supported")
```
Comment (mitechie) [inline cmd/juju/application/deploy_test.go]: why's the message coming out "logging is not available"?

Comment (mitechie) [reply cmd/juju/application/deploy_test.go]: nvm, I see that previously the name of the charm was setup as "logging".

**Comment by hmlanigan** [comment]
!!build!!

**Comment by hmlanigan** [comment]
$$merge$$

**Pull request merged**